src/components/DataTable.js

```diff
@@ -73,7 +75,23 @@ export default ajaxCaller(class DataTable extends Component {
 
     return h(Fragment, [
       !!entities && h(Fragment, [
-        childrenBefore && childrenBefore({ entities, columnSettings }),
+        div({ style: { display: 'flex' } }, [
```

Comment:
In the 'Select Data' modal when configuring a workflow, this is lacking a margin.